docs/tutorials/basics/50_control_flow.md

```diff
@@ -240,3 +240,136 @@ end
 
 Both clauses have branches with the same conditions but in a different order and they behave differently.
 The first matching condition selects which branch executes.
+
+## Loops
+
+This section delves into writing code that it will repeat until a condition (aka the `loop condition`) is met
```

Comment:
This line doesn't work: what is _it_ in "it will repeat"? I repeat my suggestion:

```suggestion
This section introduces the basics of repeated execution of code.
```